ang/civicase/ActivitiesCalendar.html

```diff
@@ -9,12 +9,20 @@
     <div class="arrow"></div>
     <div class="popover-content" ng-switch="loadingActivities">
       <civicase-activity-card-placeholder ng-switch-when="true"></civicase-activity-card-placeholder>
-      <div ng-switch-when="false" ng-repeat="activity in selectedActivites | orderBy: '-is_overdue'"
+      <div
+        ng-switch-when="false"
```

Comment:
This is just a note since I know this was not introduced by this PR, but I'm would not suggest using ng-switch when there are only two values. When I see "false" I start wondering what that value means, then I need to scroll up trying to find where the switch started to understand what's the meaning of "false" or "true".

It's better to use `ng-if="loadingActivities"` and `ng-if="!loadingActivities"`. It helps understand what's going on faster.